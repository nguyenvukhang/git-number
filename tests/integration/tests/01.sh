# untracked files

init 3
log gitnu status
