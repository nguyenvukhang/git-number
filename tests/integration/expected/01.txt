On branch main

No commits yet

Untracked files:
1	[31mfile_0001[m
2	[31mfile_0002[m
3	[31mfile_0003[m

nothing added to commit but untracked files present
